gatsby-node.js

```diff
@@ -1,5 +1,69 @@
 const path = require ('path')
 const { createFilePath } = require("gatsby-source-filesystem");
+
+const paginate = async ({graphql, actions, pathPrefix, component, reporter}) => {
+  const { errors, data } = await graphql(`
+     {
+       allMarkdownRemark {
+         totalCount
+       }
+     }
+  `);
+
+  if (errors) {
+    reporter.panicOnBuild('Error while running a GraphQL query!');
+    return;
+  }
+
```

Comment:
Revert these changes since they are in another PR to avoid conflicts.

Reply:
ok.its being handled